src/Calendar/Calendar.tsx

```diff
@@ -1,27 +1,120 @@
 import './Calendar.css';
+import {useState, ChangeEvent} from 'react';
 import { createStyles, Theme, makeStyles } from '@material-ui/core/styles';
+import InputLabel from '@material-ui/core/InputLabel';
+import FormControl from '@material-ui/core/FormControl';
+import Toolbar from '@material-ui/core/Toolbar';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+
+export const months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
 
 type CalendarPropsType = {
 };
 
 const useStyles = makeStyles((theme: Theme) =>
     createStyles({
+        calendarSelect: {
+            margin: theme.spacing(1),
+            minWidth: 120,
+        },
         content: {
             flexGrow: 1,
             padding: theme.spacing(3),
         },
     })
 );
 
-const Calendar = (props: CalendarPropsType) => {
+export const Calendar = (props: CalendarPropsType) => {
     const classes = useStyles();
+
     return (
         <div className="calendar">
             <main className={classes.content}>
-                
+                <Toolbar />
+                <CalendarMonthSelect className={classes.calendarSelect}/>
+                <CalendarYearSelect className={classes.calendarSelect}/>
             </main>
         </div>
     );
 }
 
-export default Calendar;
+type CalendarMonthSelectPropsType = {
+    className: string
+};
+const CalendarMonthSelect = (props: CalendarMonthSelectPropsType) => {
+    const currentMonth = (new Date()).getMonth();
+    const [month, setMonth] = useState(String(currentMonth));
+
+    const handleMonthChange = (event: ChangeEvent<{ value: unknown }>) => {
+        setMonth(event.target.value as string);
+    };
+
+    const selectItems = months.map((m, i) => (
```

Comment:
selectItems looks a little bit meaningless. To determine what this select contains, you have to know context. If you will have 2 selects on the page, it won't be obvious at all. Maybe monthOptions would be better?